test/util.go

```diff
@@ -128,7 +128,13 @@ func (i interrupt) Error() string {
 	return "interrupted"
 }
 
-func retry(interval, timeout time.Duration, f func() error) error {
+type retryFunc func() error
+
+func (r retryFunc) RetryT(t *testing.T, interval, timeout time.Duration) {
+	require.NoError(t, r.Retry(interval, timeout))
```

Comment:
I take it this will fail the test if the `r.Retry` function returns an error other than `nil`?

Reply:
Yes - in case if returned object will be type of error and wont be nil